deploy/README.md

```diff
@@ -1,27 +1,121 @@
+```
+
+For eg.
+```bash
+./deploy/setup.sh aoi eastus
+```
+
+[setup.sh](./setup.sh) executes tasks in 3 steps
+- installs the infrastructure using [install.sh](./install.sh) script.
+- configures the infrastructure for setting up the dependecies using [configure.sh](./configure.sh) script.
+- packages the pipeline code to imported into synapse for standalone executions using [package.sh](./package.sh) script.
+
+**Note**: After the script is completed without errors, please check the batch-account pool created is created succesfully.
+
+One can find the batch account named as `batchact<10-character-random-string>` in the resource-group `<environment-code>-orc-rg`.
+
+Go to the Batch Account and switch to the pools blade. Look for one or more pools created by the bicep template. Make sure the resizing of the pool is completed without any errors. 
+
+- Error while resizing the pools are indicated by red exclamation icon next to the pool. Most common issues causing failure are related to the VM Quota limitations.
+- Resizing may take a few minutes. Pools that are resizing are indicated by `0 -> 1` numbers under dedicated nodes column. Pools that have completed resizing should show the number of dedicated nodes. 
+
+Wait for all pools to complete resizing before moving to the next steps.
+
+## Executing Synapse Pipeline
+
+We have published the Pipeline on Synapse gallery, follow the following steps to find the Synapse pipeline and execute.
+
+- Find your Synapse workspace named as `synws<10-character-random-string>` in resource group `<environment-code>-pipeline-rg`.
+- Click on `Workspace web URL` to reach the synapse workspace portal homepage.
+- Click on `Knowledge center`.
+- Click on `Browse gallery`.
+- Click on `Pipelines` in the top menu.
+- In the search box type `space` to find the pipeline named `Spaceborne Data Analysis Master Pipeline`.
+- Click on pipeline name to select it, and press `Continue` button at the end of the page.
+
+Follow the steps in [instructions.md](./gallery/instructions.md) to configure and execute the pipeline.
+
+## Cleanup Script
+
+We have a cleanup script to cleanup the resource groups and thus the resources provisioned using the `environmentCode`.
+As discussed above the `environmentCode` is used as prefix to generate resource group names, so the cleanup-script deletes the resource groups with generated names.
+
+Execute the cleanup script as follows:
+
+```bash
+./deploy/cleanup.sh <environmentCode>
+```
+
+For eg.
+```bash
+./deploy/cleanup.sh aoi
+```
+
+If one wants not to delete any specific resource group and thus resource they can use NO_DELETE_*_RESOURCE_GROUP environment variable, by setting it to true
+
+```bash
+NO_DELETE_DATA_RESOURCE_GROUP=true
+NO_DELETE_MONITORING_RESOURCE_GROUP=true
+NO_DELETE_NETWORKING_RESOURCE_GROUP=true
+NO_DELETE_ORCHESTRATION_RESOURCE_GROUP=true
+NO_DELETE_PIPELINE_RESOURCE_GROUP=true
+./deploy/cleanup.sh <environmentCode>
+```
 
 ## AI Model
```

Comment:
Do we still need this section? @mandarinamdar @senthilkungumaraj

Reply:
Its already covered in main README, we can remove this from this deploy readme section

Reply:
@mandarinamdar we decided to remove from the main README

Reply:
In the main README, it is covered under GETTING started. I think that is repeating here and we are adding just last line for bring your own model. But bring your own model is again not covering that alone but covering additional content to use existing model in ACR/github.

Reply:
Lets call out only this part of that section

**Follow the [document](https://github.com/sushilkm/Azure-Orbital-Analytics-Samples/blob/suskuma/doc-fixes/deploy/bring-your-own-ai-model.md) to understand and use custom vision model (Hyperlink) or bring different ai-model for processing with the pipeline.** 

Also lets rename  bring-your-own-ai-model.md to **using-ai-model-in-pipeline.md or something similar**